packages/fcl/src/utils/deprecate.js

```diff
@@ -0,0 +1,5 @@
+import * as logger from "@onflow/util-logger"
+
+export const deprecate = async ({title, message, level, always}) => {
```

Comment:
This is just spitting out the logger. Does it make sense to have the message in here and you pass the args to deprecate you want to fill in so we don't have to repeat the deprecation message outside of this every time and the function name is more specific to it?